src/shared/components/main-gradient/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { ColorScheme as ThemeColor, mainStore } from '@screens/main/store/store'
+import { LinearGradient } from 'expo-linear-gradient'
+import { type FC } from 'react'
+import { View, useWindowDimensions } from 'react-native'
+
+const MainGradient: FC = () => {
+  const { height } = useWindowDimensions()
+  const storeColorScheme = mainStore.use.colorScheme()
+  const dinamicHeight = ((height / 100) * 10).toFixed()
```

Comment:
multiply by 0.1 to prevent two operations
```suggestion
  const dinamicHeight = (height * 0.1).toFixed()
```